packages/xstate-test/src/machine.ts

```diff
@@ -104,10 +109,15 @@ export function createTestModel<TMachine extends AnyStateMachine>(
   const testModel = new TestModel<StateFrom<TMachine>, EventFrom<TMachine>>(
     machine as SimpleBehavior<any, any>,
     {
-      serializeState: (state, event) => {
-        return `${serializeState(state)}${serializeTransition(state, event, {
-          serializeEvent
-        })}` as SerializedState;
+      serializeState: (state, event, prevState) => {
+        return `${serializeState(state)}${serializeTransition(
```

Comment:
If I understand correctly this one intentionally doesn't call `serializeState` with `event` and `prevState`. Could we add a comment that would explain this here?